docs/kpod-umount.1.md

```diff
@@ -0,0 +1,19 @@
+% kpod(1) kpod-umount - Unmount a working container's root filesystem.
+% Dan Walsh
+# kpod-umount "1" "July 2017" "kpod"
+
+## NAME
+kpod umount - Unmount a working container's root file system.
```

Comment:
It should get a busy error from `umount()`, but it won't be attempted if you've manually `buildah mount`ed it first, since they're also ref-counted.